src/Makevars.win

```diff
@@ -1,4 +1,5 @@
 PKG_CXXFLAGS += -DRCPP_PARALLEL_USE_TBB=1
+PKG_CXXFLAGS += $(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" -e "RcppParallel::CxxFlags()")
```

Comment:
If I understand correctly, [the output will already include the flag](https://github.com/RcppCore/RcppParallel/blob/5aa08f8b546d2fa99372d02d7b6a50344fb09ff3/R/tbb.R#L53) from line 1, so we could just remove the previous line.